tests/TestYamlTools.cpp

```diff
@@ -0,0 +1,51 @@
+#include <random>
+#include <fstream>
```

Comment:
do you need fstream? you do not read any files do you?

Reply:
Removed as well